.reuse/dep5

```diff
@@ -16,4 +16,8 @@ License: Apache-2.0
 
 Files: requirements*.txt
 Copyright: 2021 LG Electronics
-License: Apache-2.0
\ No newline at end of file
+License: Apache-2.0
+
+Files: .bumpversion.cfg
```

Comment:
@bjk7119 ,
.bumpversion.cfg seems to be the value set when using bumpversion. 

Did you copy this from another OSS? 
Didn't you see https://github.com/peritus/bumpversion and write the settings?

Reply:
@soimkim 
Yes, you are right. It's my mistake.
The contents in .bumpversion.cfg are not copy of any other OSS.

So, I changed License and copyright